source/client-side-encryption/client-side-encryption.rst

```diff
@@ -61,14 +75,25 @@ schema
    specification, `as documented in the MongoDB
    manual. <https://docs.mongodb.com/manual/reference/operator/query/jsonSchema/>`_.
 
-libmongocrypt
+libmongocrypt_
    A library, written in C, that coordinates communication,
    does encryption/decryption, caches key and schemas. `Located here <https://github.com/mongodb/libmongocrypt>`_.
 
-mongocryptd
+mongocryptd_
    A local process the driver communicates with to determine
    how to encrypt values in a command.
 
+csfle_
+   This initialism, spelled in all-lowercase, refers to the *client-side
+   field-level-encryption* dynamic library provided as part of a MongoDB
+   Enterprise distribution. It replaces mongocryptd_ as the method of
+   :ref:`marking-up a database command for encryption <subtype6.intent-to-encrypt>`.
+
+   .. seealso::
```

Comment:
Note sure if this was already mentioned but most RST features like `.. seealso` and `.. option` unfortunately do not work on Github's renderer so we will need to replace them with normal text. For example: https://github.com/mongodb/specifications/blob/9c8d7eafae935a681439113263d7a0f250923072/source/client-side-encryption/client-side-encryption.rst#csfle

Reply:
Removed those.

Aside: I have a branch that supports full HTML rendering and I'm also interested in getting a way to automatically publish such HTML pages to an easily-browse-able site (perhaps gh-pages?). Maybe someday...

Reply:
Thanks!